commit/tokenprice/processor.go

```diff
@@ -66,6 +67,9 @@ func (p *processor) Outcome(
 		p.lggr.Debugw("TokenPriceBatchWriteFrequency is set to zero, no prices will be reported")
 		return Outcome{}, nil
 	}
+	if len(aos) == 0 {
+		return Outcome{}, fmt.Errorf("no observations to process")
```

Comment:
Do we wanna return an error here or just an empty outcome? I don't really see this as an error

Reply:
You're right, this would be a problem for the discovery process specifically. On second inspection, the chainfee processor (and probably all of them) use the same pattern with no `len(aos)` check.

I think the audit issue is invalid, lets close this PR with no change.